sources/osgViewer/Viewer.js

```diff
@@ -449,7 +449,9 @@ define( [
         beginFrame: function () {
             this._startFrameTick = Timer.instance().tick();
         },
-
+        setEndFrameCallBack: function ( u ) {
```

Comment:
mm first iteration added that before the camera callback. 
Not needed here, still would find that useful in many case/experiments with code.
can remove